src/os/inc/osapi-timer.h

```diff
@@ -61,31 +61,30 @@ typedef struct
  * which is created and deleted with the timer object itself.  The internal time base
  * is configured for an OS simulated timer tick at the same interval as the timer.
  *
+ * The callback function should be declared as follows:
+ *
+ *     <tt> void timer_callback(osal_id_t timer_id); </tt>
+ *
```

Comment:
Suggestion - just reference OS_TimerCallback_t?

Reply:
Resolved in 1e0025bf